test_cfms/c_horiz_interp/test_horiz_interp_new.c

```diff
@@ -223,3 +270,196 @@ int main()
     return EXIT_SUCCESS;
 }
 
+
+int test_bilinear_new(int domain_id)
+{
+    // domain info
+    int isc, iec, jsc, jec;
+    int xsize_c = 0;
+    int ysize_c = 0;
+    int whalo = WHALO;
+    int ehalo = EHALO;
+    int shalo = SHALO;
+    int nhalo = NHALO;
+    int *xmax_size  = NULL;
+    int *ymax_size  = NULL;
+    int *tile_count = NULL;
+    int *position   = NULL;
+    bool *x_is_global = NULL;
+    bool *y_is_global = NULL;
+    
+    // grid data
+    double *lat_in_1D, *lon_in_1D;
+    double *lat_in_2D, *lon_in_2D;
+    double *lat_out_1D, *lon_out_1D;
+    double *lat_out_2D, *lon_out_2D;
+    double *lon_src_1d, *lat_src_1d;
+    double *lon_dst_1d, *lat_dst_1d;
+    double dlon_src, dlat_src, dlon_dst, dlat_dst;
+
+    double lon_src_beg = 0.;
+    double lon_src_end = 360.;
+    double lat_src_beg = -90.;
+    double lat_src_end = 90.;
+    double lon_dst_beg = 0.;
+    double lon_dst_end = 360.;
+    double lat_dst_beg = -90.;
+    double lat_dst_end = 90.;
+    double D2R = 3.14/180.;
```

Comment:
can get DEG_TO_RAD with `#include "c_constants.h"`

Reply:
This should be fixed now too